tests/core/string/test_translation_server.h

```diff
@@ -0,0 +1,136 @@
+/**************************************************************************/
+/*  test_translation_server.h                                             */
+/**************************************************************************/
+/*                         This file is part of:                          */
+/*                             GODOT ENGINE                               */
+/*                        https://godotengine.org                         */
+/**************************************************************************/
+/* Copyright (c) 2014-present Godot Engine contributors (see AUTHORS.md). */
+/* Copyright (c) 2007-2014 Juan Linietsky, Ariel Manzur.                  */
+/*                                                                        */
+/* Permission is hereby granted, free of charge, to any person obtaining  */
+/* a copy of this software and associated documentation files (the        */
+/* "Software"), to deal in the Software without restriction, including    */
+/* without limitation the rights to use, copy, modify, merge, publish,    */
+/* distribute, sublicense, and/or sell copies of the Software, and to     */
+/* permit persons to whom the Software is furnished to do so, subject to  */
+/* the following conditions:                                              */
+/*                                                                        */
+/* The above copyright notice and this permission notice shall be         */
+/* included in all copies or substantial portions of the Software.        */
+/*                                                                        */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,        */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF     */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. */
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY   */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,   */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE      */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
+/**************************************************************************/
+
+#ifndef TEST_TRANSLATION_SERVER_H
+#define TEST_TRANSLATION_SERVER_H
+
+#include "core/string/translation.h"
+
+#include "tests/test_macros.h"
+
+namespace TestTranslationServer {
+TEST_CASE("[TranslationServer] Translation operations") {
+	Ref<Translation> t = memnew(Translation);
+	t->set_locale("uk");
+	t->add_message("Good Morning", String::utf8("Добрий ранок"));
+
+	TranslationServer *ts = TranslationServer::get_singleton();
+
+	int l_count_before = ts->get_loaded_locales().size();
+	ts->add_translation(t);
+	int l_count_after = ts->get_loaded_locales().size();
+	// Newly created Translation object should be added to the list, so the counter should increase, too
```

Comment:
```suggestion
	// Newly created Translation object should be added to the list, so the counter should increase, too.
```

Comments should start with a capital and end with a period, same elsewhere